nixos/modules/system/boot/loader/limine/limine-install.py

```diff
@@ -0,0 +1,413 @@
+#!@python3@/bin/python3 -B
+
+from dataclasses import dataclass
+from typing import Any, Dict, List, Optional, Tuple
+
+import datetime
+import hashlib
+import json
+import os
+import psutil
+import re
+import shutil
+import subprocess
+import sys
+import tempfile
+import textwrap
+
+
+limine_dir = None
+root_fs_uuid = None
+can_use_direct_paths = False
+install_config = json.load(open('@configPath@', 'r'))
+
+
+def config(*path: List[str]) -> Optional[Any]:
+    result = install_config
+    for component in path:
+        result = result[component]
+    return result
+
+
+def get_system_path(profile: str = 'system', gen: Optional[str] = None, spec: Optional[str] = None) -> str:
+    if profile == 'system':
+        result = '/nix/var/nix/profiles/system'
+    else:
+        result = f'/nix/var/nix/profiles/system-profiles/{profile}'
+
+    if gen is not None:
+        result += f'-{gen}-link'
+
+    if spec is not None:
+        result = os.path.join(result, 'specialisation', spec)
+
+    return result
+
+
+def get_profiles() -> List[str]:
+    profiles_dir = '/nix/var/nix/profiles/system-profiles/'
+    dirs = os.listdir(profiles_dir) if os.path.isdir(profiles_dir) else []
+
+    return [path for path in dirs if not path.endswith('-link')]
+
+
+def get_gens(profile: str = 'system') -> List[Tuple[int, List[str]]]:
+    nix_env = os.path.join(config('nixPath'), 'bin', 'nix-env')
+    output = subprocess.check_output([
+        nix_env, '--list-generations',
+        '-p', get_system_path(profile),
+        '--option', 'build-users-group', '',
+    ], universal_newlines=True)
+
+    gen_lines = output.splitlines()
+    gen_nums = [int(line.split()[0]) for line in gen_lines]
+
+    return [gen for gen in gen_nums][-config('maxGenerations'):]
+
+
+def is_encrypted(device: str) -> bool:
+    for name, _ in config('luksDevices').items():
+        if os.readlink(f'/dev/mapper/{name}') == os.readlink(device):
+            return True
+
+    return False
+
+
+def is_fs_type_supported(fs_type: str) -> bool:
+    return fs_type.startswith('vfat')
+
+
+def get_copied_path_uri(path: str, target: str) -> str:
+    result = ''
+
+    if can_use_direct_paths:
+        result = f'uuid://{root_fs_uuid}{path}'
+    else:
+        package_id = os.path.basename(os.path.dirname(path))
+        suffix = os.path.basename(path)
+        dest_file = f'{package_id}-{suffix}'
+        dest_path = os.path.join(limine_dir, target, dest_file)
+
+        if not os.path.exists(dest_path):
+            copy_file(path, dest_path)
+
+        path_with_prefix = os.path.join('/limine', target, dest_file)
+        result = f'boot()://{path_with_prefix}'
```

Comment:
```suggestion
        result = f'boot():{path_with_prefix}'
```
Likely another remnant from the old config format, this one seemingly doesn't affect anything however in the new config format, a single `/` is enough.